src/ShopBundle/objects/TCMSSmartURLHandler/TCMSSmartURLHandler_ShopPaymentSofortueberweisungAPI.class.php

```diff
@@ -16,7 +16,10 @@
 
 /**
  * takes sofortueberweisung return urls and maps them to chameleon urls.
-/**/
+ *
+ * @psalm-suppress UndefinedPropertyAssignment
+ * @FIXME Writing data into `$OURLData` when there is no magic `__set` method for them defined.
```

Comment:
In this class, there is a lot of writing into 'magic' properties of `TCMSSmartURLData` - however, those properties are not writable.

(Sorry for repeating this comment a lot)